src/main/java/edu/wpi/first/pathweaver/FxUtils.java

```diff
@@ -9,12 +11,14 @@
 import javafx.scene.control.TreeView;
 
 public final class FxUtils { // NOPMD util class name
-  private static final PseudoClass[] SUBCHILD_SELECTORS = new PseudoClass[8]; // NOPMD
+  private static final List<PseudoClass> SUBCHILD_SELECTORS; // NOPMD
 
   static {
-    for (int i = 0; i < SUBCHILD_SELECTORS.length; i++) {
-      SUBCHILD_SELECTORS[i] = PseudoClass.getPseudoClass("subchild" + i);
+    PseudoClass[] selectors = new PseudoClass[8];
+    for (int i = 0; i < selectors.length; i++) {
+      selectors[i] = PseudoClass.getPseudoClass("subchild" + i);
     }
+    SUBCHILD_SELECTORS = List.of(selectors);
```

Comment:
Use `Collections.unmodifiableList`

Reply:
Why? I originally did use that and IntelliJ suggested replacing it with `List.of`. It seems that both are immutable, and using `List.of` doesn't require calling `Arrays.asList` first.

Reply:
`List.of` is the preferred method starting in Java 9

Reply:
Oh.  You are right